### PR TITLE
translate hpo_train.md and perf_hardware.md to chinese

### PR DESCRIPTION
**Pull request by jiaqiw09**
# What does this PR do?

Part of  #26803 

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [x] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests? _not necessary_

## Who can review?
@stevhliu

### CODE REVIEW

File: docs/source/zh/perf_hardware.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+- 更多的 GPU
+- 更多的 CPU 和 NVMe（通过[DeepSpeed-Infinity](main_classes/deepspeed#nvme-support)实现）
+
+让我们从只有一块GPU的情况开始。
+
+### 供电和散热
+
+如果您购买了昂贵的高端GPU，请确保为其提供正确的供电和足够的散热。
+
+**供电**
```
Comment (stevhliu) [inline docs/source/zh/perf_hardware.md:41]:
```suggestion
**供电**:
```

File: docs/source/zh/perf_hardware.md [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+
+其他一些卡可能使用PCI-E-12针连接器，这些连接器可以提供最多500-600W的功率。
+
+低端卡可能使用6针连接器，这些连接器可提供最多75W的功率。
+
+此外，您需要选择具有稳定电压的高端电源。一些质量较低的电源可能无法为卡提供所需的稳定电压以发挥其最大性能。
+
+当然，电源还需要有足够的未使用的瓦数来为卡供电。
+
+**散热：**
```
Comment (stevhliu) [inline docs/source/zh/perf_hardware.md:55]:
```suggestion
**散热**:
```

File: docs/source/zh/perf_hardware.md [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+因此，第一个报告`NV2`告诉我们GPU通过2个NVLink互连，而第二个报告`PHB`展示了典型的消费者级PCIe+Bridge设置。
+
+检查你的设置中具有哪种连接类型。其中一些会使卡之间的通信更快（例如NVLink），而其他则较慢（例如PHB）。
+
+根据使用的扩展解决方案的类型，连接速度可能会产生重大或较小的影响。如果GPU很少需要同步，就像在DDP中一样，那么较慢的连接的影响将不那么显著。如果GPU经常需要相互发送消息，就像在ZeRO-DP中一样，那么更快的连接对于实现更快的训练变得非常重要。
+
+
+#### NVlink
+
+NVLink是由Nvidia开发的一种基于线缆的串行多通道近程通信链接。
```
Comment (stevhliu) [inline docs/source/zh/perf_hardware.md:107]:
```suggestion
[NVLink](https://en.wikipedia.org/wiki/NVLink)是由Nvidia开发的一种基于线缆的串行多通道近程通信链接。
```

**Review by stevhliu — APPROVED** [review]
Thanks for translating these 🙂

**Comment by jiaqiw09** [comment]
@stevhliu 

hi, thansk for your review. I just fix problems mentioned in reviews.

Best

**Pull request merged**